lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
@weerd Im adding northstar ID because I needed that for signups, which I was not getting before.

Im adding drupal ID because it was confusing  what "Id" was